opm/simulators/flow/BlackoilModelEbos.hpp

```diff
@@ -642,7 +643,10 @@ namespace Opm {
                                                     // residual
 
             // if the solution is updated, the intensive quantities need to be recalculated
-            ebosSimulator_.model().invalidateAndUpdateIntensiveQuantities(/*timeIdx=*/0);
+            {
+                OPM_TIMEBLOCK(invalidateAndUpdateIntensiveQuantities);
```

Comment:
Better to have this block name in the opm-models implementation, as that may be called from other places than here.